docs/articles/examples/create-your-own-examples.mdx

```diff
@@ -0,0 +1,120 @@
+# Custom Test Workflow Examples
+
+This guide shows two ways to publish your own Test Workflow Templates so they appear in the Dashboard wizard and can be reused across teams.
+
+- Use kubectl for quick, ad-hoc rollout by a platform team.
+- Use a small Helm chart to version and ship a curated catalog of templates.
+
+> See the Examples & Guides catalog and Workflow Templates docs for built‑in templates.
+
+## Prerequisite: show Official Templates in the Wizard
+
+If you also want the built‑in templates to be created and visible in the wizard, make sure the Helm values include:
+
+```yaml
+global:
+  testWorkflows:
+    createOfficialTemplates: true
+```
+
+This flag belongs to the Testkube chart values and is applied during install/upgrade.
+
+## Option A: Apply a template with kubectl
+
+1) Create a `TestWorkflowTemplate` with the required labels so it shows up in the wizard:
```

Comment:
Could we add a section that explains the "required labels" and how they map to the different sections in the wizard?